docs/Catalog/Admin.md

```diff
@@ -64,10 +69,78 @@ The resulting permission set is equivalent to a union of all permissions
 provided by the policies attached to that role.
 
 
+### Extending built-in roles
+
+The initial "Source=Custom" roles defined by Quilt (`ReadQuiltBucket`, `ReadWriteQuiltBucket`) are locked to AWS ARNs, and thus cannot be extended by Quilt policies.  If you want a "Source=Quilt" UI-managed role that automatically has access to all registered buckets, you need to first import those specific polices.
+
+
+#### A. Create a Source=Quilt role
+
+For concreteness, we will create a new `UserReadQuiltBucket` read-only role to which we can add Quilt policies.
+
+1. Login to your Quilt instance at, e.g. https://quilt.mycompany.com
+2. Click on "Admin Settings" in the upper right, under your Profile name
+3. Scroll down to the "Roles" section on the bottom
+4. Click "+" to define a new role
+5. Name it `UserReadQuiltBucket`
+6. Save
+
+You can also create a `UserReadWriteQuiltBucket` if you want grant full permissions
+
+#### B. Display existing policies in AWS Console
+
+While still in the Roles section of Admin Settings:
+
+1. Find 'Custom' role "ReadWriteQuiltBucket"
```

Comment:
There's no reason to rely on this and it will only confuse people. Until we fix Source=Custome roles please just ask people to create fresh QUILT ROLES and add the desired buckets there. That will work.